boring/src/ssl/mod.rs

```diff
@@ -2748,6 +2764,21 @@ impl SslRef {
             .expect("invalid default server curves list");
     }
 
+    /// Returns the [`SslCurve`] used for this `SslRef`.
+    ///
+    /// This corresponds to [`SSL_get_curve_id`]
+    ///
+    /// [`SSL_get_curve_id`]: https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_get_curve_id
+    #[cfg(not(feature = "kx-safe-default"))]
+    pub fn get_curve(&self) -> Option<SslCurve> {
```

Comment:
Other getters don't use a `get` prefix.

```suggestion
    pub fn curve(&self) -> Option<SslCurve> {
```